mmhuman3d/core/cameras/cameras.py

```diff
@@ -259,7 +264,7 @@ def get_default_projection_matrix(cls, ):
         """
         raise NotImplementedError()
 
-    def to_screen_(self, **kwargs):
+    def to_screen_(self, **kwargs) -> 'MMCamerasBase':
         """Convert to screen inplace."""
         if self.in_ndc():
             if self.get_image_size() is None:
```

Comment:
If `image_size` has been passed to `to_screen_`, it can be inferred that the caller want's to override `image_size` anyway. Shouldn't we take `image_size` as the first choice, or raise a warning to tell users that your `image_size` is not in use.

Reply:
I think a warning is ok